models/topic.model.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection");
+
+exports.selectTopics = () => {
+  return db.query(`SELECT * FROM topics;`);
```

Comment:
If you wanted a tidier controller, you can add a .then block in here to return the rows.